packages/assets-controllers/package.json

```diff
@@ -62,6 +63,7 @@
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/ethjs-provider-http": "^0.3.0",
     "@metamask/keyring-api": "^3.0.0",
+    "@types/bn.js": "^5.1.5",
```

Comment:
When I run `yarn build` I see this line in `packages/assets-controllers/dist/AssetsContractController.d.ts`:

``` typescript
import type BN from 'bn.js';
```

For this reason I think we need to have this in `dependencies` and not `devDependencies`, otherwise TypeScript consumers will need to add this themselves.